src/haxelib/client/Args.hx

```diff
@@ -276,6 +278,7 @@ class Args {
 		addCommand(Setup, "set the haxelib repository path", Miscellaneous);
 		addCommand(NewRepo, "create a new local repository", Miscellaneous);
 		addCommand(DeleteRepo, "delete the local repository", Miscellaneous);
+		addCommand(Lock, "output an hxml file with all the installed libraries", Miscellaneous);
```

Comment:
This doesn't seem like the most relevant description...

Reply:
Is "output an hxml file with installation instructions for installed libraries" any better?

Reply:
Uh, barely...

I'd expect a lock command to mention, you know, locking? The fact that HXML files are involved doesn't even seem particularly important.

Reply:
Well that's the thing, it doesn't lock anything. Maybe the issue is more with the command name?

Reply:
Are you saying we have a lock command that creates a lock file that doesn't lock anything? I'm not sure I understand what's happening here...

Reply:
Oh so that's what happens here... in this case I agree that "lock" is not a good name, and the description should probably end with "for use with `haxelib install`".

Reply:
Hopefully this version is better (check updated PR description)